internal/hooks/spec.go

```diff
@@ -0,0 +1,70 @@
+package hooks
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	oci "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// Note: The below type definition as well as constants have been copied from
+// https://github.com/opencontainers/runc/blob/master/libcontainer/configs/config.go.
+// This is done to not introduce a direct dependency on runc, which would complicate
+// integration with windows.
```

Comment:
is the goal of this package to house all potential hook binaries we may want for lcow or wcow then?

Reply:
UpdatePathEnv is still linux specific though

Reply:
yeah, the goal was to have something generic which will inject hooks into spec. you're right about `UpdatePathEnv`, I'll probably move it down to linux specific code path, until we have a use case for it on windows.